presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/env/common/Standard.java

```diff
@@ -117,4 +128,27 @@ public static DockerContainer createPrestoContainer(DockerFiles dockerFiles, Pat
                 .waitingFor(Wait.forLogMessage(".*======== SERVER STARTED ========.*", 1))
                 .withStartupTimeout(Duration.ofMinutes(5));
     }
+
+    public static void enablePrestoJavaDebugger(DockerContainer container, int debugPort)
+    {
+        try {
+            FileAttribute<Set<PosixFilePermission>> rwx = PosixFilePermissions.asFileAttribute(PosixFilePermissions.fromString("rwxrwxrwx"));
+            Path script = Files.createTempFile("enable-java-debugger", ".sh", rwx);
+            Files.writeString(
+                    script,
+                    format(
```

Comment:
This format is too complicated. You mix concatenation and format. Also you use format to concatenate two string literals. 

Let's inline and use only concatenation.

Also I know that some people would extract a bash script from that.

Reply:
I use `format` to formulate how i inject JVM arg into config.properties file.
I use concatenation to produce the JVM arg (since the variable part is last, `format` is not applicable).
Since the bash is oneliner, extracting it to a file would be an overkill.
Let me keep this.